.ci/go-no-os-exit.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+go_packages=.
+
+candidates=`go list -f '{{.Dir}}/*.go' $go_packages`
+for f in $candidates; do
+	filename=`basename $f`
+	# skip exit.go where, the only file we should call os.Exit() from.
+	[[ $filename == "exit.go" ]] && continue
+	# skip main_test.go
+	[[ $filename == "main_test.go" ]] && continue
+	files="$f $files"
+done
+
+if grep -n os.Exit $files; then
```

Comment:
Why -q? we don't want it to be quiet but tell us where offending os.Exit is (file/line)